### ISSUE TITLE
data-turbo-cache deprecated

### ISSUE THREAD

**Issue by intersecato** [php-flasher/php-flasher#174]
Getting an error "The [data-turbo-cache=false] selector is deprecated and will be removed in a future version. Use [data-turbo-temporary] instead" when using flasher-toastr.

**Comment by yoeunes** [comment]
hello @intersecato, thanks for reporting the issue, can you show me the steps to reproduce the issue please ?

**Comment by intersecato** [comment]
honestly there isn't much to explain. data-turbo-cache is deprecated and from what I see it is contained within 
flasher-toastr/Resources/assets/flasher-toastr.min.js

**Comment by yoeunes** [comment]
Thank you for the details. I'm releasing v2.0 next week and will include a fix for this issue. For now, you can clear notifications with Turbo using this snippet:

```javascript
document.addEventListener('turbo:before-cache', () => {
    // 
});
```

**Comment by intersecato** [comment]
Hi @yoeunes 
I know it wouldn't be the right repository but I also wanted to point out another deprecation:

_User Deprecated: Method "Symfony\Component\HttpKernel\Bundle\Bundle::boot()" might add "void" as a native return type declaration in the future. Do the same in child class "Flasher\Symfony\FlasherSymfonyBundle" now to avoid errors or add an explicit @return annotation to suppress this message._

**Comment by yoeunes** [comment]
Hi @intersecato,

Thank you for bringing up the Symfony deprecation notice as well. Actually, you're indeed in the right place since the source code is housed in this repository as a monorepo. you can see the updated method here: https://github.com/php-flasher/php-flasher/blob/2.x/src/Symfony/FlasherSymfonyBundle.php#L18

If you have the time, I'd greatly appreciate it if you could review other parts of v2.x branch and let me know if you see any other potential improvements or deprecations. Thanks a lot for your contribution 🙏🏼

**Comment by intersecato** [comment]
Hi, I saw the v2.0 release but I see that the repos https://github.com/php-flasher/flasher-symfony and https://github.com/php-flasher/flasher-toastr-symfony are still at version 1.15.14, is this normal?

**Comment by yoeunes** [comment]
Hi @intersecato 👋🏼,

Thanks for reaching out! Yes, I've released v2.0.0 across all the related repositories, including https://github.com/php-flasher/flasher-symfony and https://github.com/php-flasher/flasher-toastr-symfony. They should be correctly tagged as v2.0.0. You can also verify this on Packagist at the following links:

- https://packagist.org/packages/php-flasher/flasher-symfony
- https://packagist.org/packages/php-flasher/flasher-toastr-symfony

Could you please try installing the v2.0.0 and let me know if everything works as expected?

Thanks!

**Comment by yoeunes** [comment]
Hi @intersecato 👋🏼,

I’m closing this issue due to inactivity. If you’re still experiencing this problem with the latest version of PHPFlasher, please feel free to reopen it or open a new issue.

Thanks.
